chatsky/utils/db_benchmark/basic_config.py

```diff
@@ -177,27 +177,23 @@ def context_updater(self, context: Context) -> Optional[Context]:
 basic_configurations = {
     "large-misc": BasicBenchmarkConfig(
         from_dialog_len=1,
-        to_dialog_len=50,
+        to_dialog_len=26,
         message_dimensions=(3, 5, 6, 5, 3),
         misc_dimensions=(2, 4, 3, 8, 100),
     ),
     "short-messages": BasicBenchmarkConfig(
-        from_dialog_len=500,
-        to_dialog_len=550,
         message_dimensions=(2, 30),
         misc_dimensions=(0, 0),
     ),
     "default": BasicBenchmarkConfig(),
-    "large-misc--long-dialog": BasicBenchmarkConfig(
-        from_dialog_len=500,
-        to_dialog_len=550,
+    "large-misc-long-dialog": BasicBenchmarkConfig(
         message_dimensions=(3, 5, 6, 5, 3),
         misc_dimensions=(2, 4, 3, 8, 100),
     ),
```

Comment:
Maybe make it `from_dialog_len=50`, `to_dialog_len=75`?
Otherwise the name "large-misc-long-dialog" is only partially true.